.env.test

```
@@ -2,4 +2,5 @@ NETWORK="regtest"
 DOMAINS={"12cash.dev": "id123"}
 PROVIDER="cloudflare"
 CF_TOKEN=""
-JWT_SECRET="abc123"
\ No newline at end of file
+JWT_SECRET="abc123"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)